cps/main.c

```diff
@@ -575,6 +570,27 @@ static int
 assign_cps_queue_ids(struct cps_config *cps_conf)
 {
 	int ret;
+	/* Take the packets created for processing requests from mailbox. */
+	unsigned int total_pkt_burst = cps_conf->total_pkt_burst +
+		cps_conf->mailbox_burst_size;
```

Comment:
We must add `cps_conf->front_max_pkt_burst` and `cps_conf->back_max_pkt_burst` if the back is enabled to account for the packets of the KNI interfaces. Update the comment to explain it.